client/graphql/mutations/changeProfilePicture.graphql

```diff
@@ -0,0 +1,5 @@
+mutation changeProfilePicture($picture: String!) {
+  changeProfilePicture(picture: $picture) {
+    ok
```

Comment:
If you changed this query to return the user with id and profile picture, then you wouldn't need to refetch those other queries like user profile.

Reply:
I have gone around one of the refetches by returning the user, refetching the profile is not needed anymore, but buildList is in another component, so I probably should keep that refetch in there? When I return the user I cannot get and repopulate userCustomSets, can I?